action.yml

```diff
@@ -1,5 +1,10 @@
 name: 'codenotify'
 description: 'Notify subscribed users and groups of changes.'
+inputs:
+  filename:
+    description: 'Filename in which file subscribers are defined'
+    required: false
+    default: 'CODENOTIFY'
```

Comment:
Can you elaborate on why you think that solution would be cleaner? And what's "exotic" about this?

I see that it would be more flexible (in theory you could use it to set _any_ option) but at least from the GitHub actions side, it doesn't seem any cleaner to me.

Note that there are basically two code paths that determine how the options get set, and currently when this is run in GitHub actions, the command line arguments are currently ignored:

```
func getOptions(stdout io.Writer, args []string) (*options, error) {
	if os.Getenv("GITHUB_ACTIONS") == "true" {
		return githubActionOptions()
	}
	return cliOptions(stdout, args)
}
```

I might have done this differently, but I'm hesitant to change that here given the difficulty of testing those changes. I'm not sure it's possible to do it how you suggest without doing a more substantial refactor of the option handling.

Reply:
Your comment triggered me to read the docs on `runs` with Docker, to see if I missed something. I did miss that you cannot pass a `cmd` field as I initially thought. I thought that we had more control over how the container is run. 

The `input` is transformed into an env var automatically, which indeed makes it the simplest way here. With now that in mind, I think that making the env var having a default value of `CODENOTIFY` as I mentioned in my other comment is indeed better.